.github/workflows/release.yml

```diff
@@ -167,3 +167,40 @@ jobs:
           file: target/release/maturin.tar.gz
           tag: ${{ github.ref }}
           prerelease: ${{ contains(github.ref, 'alpha') || contains(github.ref, 'beta') }}
+
+  release-gnu-pypi:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [
+          { target: "s390x-unknown-linux-gnu", image: "messense/manylinux2014-cross:s390x", manylinux: "2014" },
```

Comment:
Rust doesn't currently distribute `s390x-unknown-linux-musl` target stdlib, so here we use GNU target instead.